python/descriptor_pool.c

```diff
@@ -62,10 +72,17 @@ static int PyUpb_DescriptorPool_Clear(PyUpb_DescriptorPool* self) {
   return 0;
 }
 
+PyObject* PyUpb_DescriptorPool_Get(const upb_symtab *symtab) {
+  PyObject *pool = PyUpb_ObjCache_Get(symtab);
```

Comment:
you got some pointers wandering from east to west, run clang-format

Reply:
Done.